### DISCUSSION TITLE
Just curious - what features are removed do to SMC calls in App Store?

### DISCUSSION THREAD

**Discussion by jevonmao** [gao-sun/eul#148] [General]
What kind of SMC calls does App Store not allow? What's the actual difference between Github version software and App Store version software? Just curious

**Comment by gao-sun** [comment]
According to what I discovered on the Internet, Apple doesn't like any kind of SMC calls. Specifically in eul, temperatures and fan speeds.

The access to system paths is not allowed as well and they won't work if you enable "App Sandbox" (which is required by AppStore). So you can see network component in the AppStore version but it always shows 0.
